addons/pos_hr/static/tests/tours/PosHrTour.js

```diff
@@ -107,3 +107,20 @@ registry.category("web_tour.tours").add("CashierStayLogged", {
             PosHr.loginScreenIsShown(),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("CashierCanSeeProductInfo", {
+    test: true,
+    steps: () =>
+        [
+            PosHr.loginScreenIsShown(),
+            PosHr.clickLoginButton(),
+            SelectionPopup.has("Mitchell Admin", { run: "click" }),
+            Dialog.confirm("Open session"),
+            {
+                trigger: '.product-information-tag:first',
+                content: 'Open the Info popup of the first product',
+                run: 'click',
+            },
+            Dialog.confirm("Ok"),
+        ].flat(),
+});
```

Comment:
You need to add a newline at the end of the file. Depending on your IDE there should be an option to automatically add it when you save a file